src/core/Akka.FSharp.Tests/RemoteSpecs.fs

```diff
@@ -0,0 +1,67 @@
+﻿module Akka.FSharp.Tests.RemoteSpecs
+open Akka.Actor
+open Xunit
+open Xunit.Abstractions
+open Akka.FSharp
+open Akka.Remote
+open Akka.TestKit
+
+let remoteConfig port = 
+        $"""
+        akka {{ 
+            actor {{
+                ask-timeout = 5s
+                provider = remote
+            }}
+            remote {{
+                dot-netty.tcp {{
+                    port = %i{port}
+                    hostname = localhost
+                }}
+            }}
+        }}
+        """
+        |> Configuration.parse
+
+let getAddress (actorSystem:ActorSystem) =
+    let es = actorSystem :?> ExtendedActorSystem
+    es.Provider.DefaultAddress
+    
+type TestUnion = 
+    | A of string
+    | B of int * string
+
+type TestUnion2 = 
+    | C of string * TestUnion
+    | D of int
+
+type RemoteSpecs(output:ITestOutputHelper) as this =
+    inherit AkkaSpec((remoteConfig 0), output)
```

Comment:
A little convoluted, but this is how you can pass through the XUnit `ITestOutputHelper` and the `AkkaSpec` / `TestKit` base members into an F# test